pkg/ovncontroller/network.go

```diff
@@ -25,8 +25,8 @@ import (
 	ovnv1 "github.com/openstack-k8s-operators/ovn-operator/api/v1beta1"
 )
 
-// CreateAdditionalNetworks - creates network attachement definitions based on the provided mappings
-func CreateAdditionalNetworks(
+// CreateOrUpdateAdditionalNetworks - creates network attachement definitions based on the provided mappings
```

Comment:
while you are here anyway, perhaps: `attachment`; also - should this text reflect the new goal of the function - to create **and update**?

Reply:
Done